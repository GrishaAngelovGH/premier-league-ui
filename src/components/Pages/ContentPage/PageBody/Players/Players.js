import { players } from 'images'
import './Players.scss'

const Players = () => (
    <div className='row no-gutters vh-100 players-bg'>
        <div className='col-md-12'>
            <img src={players} className='img-fluid' />
        </div>
    </div>
)

export default Players