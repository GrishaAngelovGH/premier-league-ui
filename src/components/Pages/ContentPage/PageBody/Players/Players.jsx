const players = [
	{ name: "Erling Haaland", team: "Manchester City", position: "Forward", goals: 27 },
	{ name: "Mohamed Salah", team: "Liverpool", position: "Forward", goals: 18 },
	{ name: "Son Heung-Min", team: "Tottenham Hotspur", position: "Forward", goals: 17 },
	{ name: "Jarrod Bowen", team: "West Ham United", position: "Forward", goals: 16 },
	{ name: "Jean-Philippe Mateta", team: "Crystal Palace", position: "Forward", goals: 16 },
	{ name: "Alexander Isak", team: "Newcastle United", position: "Forward", goals: 21 },
	{ name: "Phil Foden", team: "Manchester City", position: "Midfielder", goals: 19 },
	{ name: "Dominic Solanke", team: "Bournemouth", position: "Forward", goals: 19 },
	{ name: "Ollie Watkins", team: "Aston Villa", position: "Forward", goals: 19 },
	{ name: "Bryan Mbuemo", team: "Brentford", position: "Forward", goals: 16 },
	{ name: "Matheus Cunha", team: "Wolverhampton Wanderers", position: "Forward", goals: 14 },
	{ name: "Ryan Gravenberch", team: "Liverpool", position: "Midfielder", goals: 8 },
	{ name: "Bruno Fernandes", team: "Manchester United", position: "Midfielder", goals: 12 },
	{ name: "Kevin De Bruyne", team: "Manchester City", position: "Midfielder", goals: 10 },
	{ name: "Morgan Gibbs-White", team: "Nottingham Forest", position: "Midfielder", goals: 10 },
	{ name: "Anthony Gordon", team: "Newcastle United", position: "Midfielder", goals: 10 },
	{ name: "Pascal GroÃŸ", team: "Brighton & Hove Albion", position: "Midfielder", goals: 8 },
	{ name: "Brennan Johnson", team: "Tottenham Hotspur", position: "Forward", goals: 9 },
]

const Players = () => (
	<div>
		<div className="container mx-auto p-4">
			<div className="grid grid-col-1 md:grid-cols-3 gap-4">
				{
					players.map((v, i) => (
						<div key={i} className="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500">
							<div>Name: {v.name}</div>
							<div>Team: {v.team}</div>
							<div>Position: {v.position}</div>
							<div>Goals: {v.goals}</div>
						</div>
					))
				}
			</div>
		</div>
	</div>
)

export default Players