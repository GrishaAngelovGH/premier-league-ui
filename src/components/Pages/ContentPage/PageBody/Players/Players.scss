.players-bg {
    background: linear-gradient(to bottom, #6457C6, purple);
}