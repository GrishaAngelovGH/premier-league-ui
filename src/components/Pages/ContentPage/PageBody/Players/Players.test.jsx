import { render } from '@testing-library/react'

import Players from './Players'

test('should render Players component', () => {
    const view = render(<Players />)

    expect(view).toMatchSnapshot()
})