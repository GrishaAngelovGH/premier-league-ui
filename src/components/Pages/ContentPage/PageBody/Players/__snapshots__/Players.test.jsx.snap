// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render Players component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="container mx-auto p-4"
        >
          <div
            class="grid grid-col-1 md:grid-cols-3 gap-4"
          >
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Erling Haaland
              </div>
              <div>
                Team: 
                Manchester City
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                27
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Mohamed Salah
              </div>
              <div>
                Team: 
                Liverpool
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                18
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Son Heung-Min
              </div>
              <div>
                Team: 
                Tottenham Hotspur
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                17
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Jarrod Bowen
              </div>
              <div>
                Team: 
                West Ham United
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                16
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Jean-Philippe Mateta
              </div>
              <div>
                Team: 
                Crystal Palace
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                16
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Alexander Isak
              </div>
              <div>
                Team: 
                Newcastle United
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                21
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Phil Foden
              </div>
              <div>
                Team: 
                Manchester City
              </div>
              <div>
                Position: 
                Midfielder
              </div>
              <div>
                Goals: 
                19
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Dominic Solanke
              </div>
              <div>
                Team: 
                Bournemouth
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                19
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Ollie Watkins
              </div>
              <div>
                Team: 
                Aston Villa
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                19
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Bryan Mbuemo
              </div>
              <div>
                Team: 
                Brentford
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                16
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Matheus Cunha
              </div>
              <div>
                Team: 
                Wolverhampton Wanderers
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                14
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Ryan Gravenberch
              </div>
              <div>
                Team: 
                Liverpool
              </div>
              <div>
                Position: 
                Midfielder
              </div>
              <div>
                Goals: 
                8
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Bruno Fernandes
              </div>
              <div>
                Team: 
                Manchester United
              </div>
              <div>
                Position: 
                Midfielder
              </div>
              <div>
                Goals: 
                12
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Kevin De Bruyne
              </div>
              <div>
                Team: 
                Manchester City
              </div>
              <div>
                Position: 
                Midfielder
              </div>
              <div>
                Goals: 
                10
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Morgan Gibbs-White
              </div>
              <div>
                Team: 
                Nottingham Forest
              </div>
              <div>
                Position: 
                Midfielder
              </div>
              <div>
                Goals: 
                10
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Anthony Gordon
              </div>
              <div>
                Team: 
                Newcastle United
              </div>
              <div>
                Position: 
                Midfielder
              </div>
              <div>
                Goals: 
                10
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Pascal Groß
              </div>
              <div>
                Team: 
                Brighton & Hove Albion
              </div>
              <div>
                Position: 
                Midfielder
              </div>
              <div>
                Goals: 
                8
              </div>
            </div>
            <div
              class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
            >
              <div>
                Name: 
                Brennan Johnson
              </div>
              <div>
                Team: 
                Tottenham Hotspur
              </div>
              <div>
                Position: 
                Forward
              </div>
              <div>
                Goals: 
                9
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="container mx-auto p-4"
      >
        <div
          class="grid grid-col-1 md:grid-cols-3 gap-4"
        >
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Erling Haaland
            </div>
            <div>
              Team: 
              Manchester City
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              27
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Mohamed Salah
            </div>
            <div>
              Team: 
              Liverpool
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              18
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Son Heung-Min
            </div>
            <div>
              Team: 
              Tottenham Hotspur
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              17
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Jarrod Bowen
            </div>
            <div>
              Team: 
              West Ham United
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              16
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Jean-Philippe Mateta
            </div>
            <div>
              Team: 
              Crystal Palace
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              16
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Alexander Isak
            </div>
            <div>
              Team: 
              Newcastle United
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              21
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Phil Foden
            </div>
            <div>
              Team: 
              Manchester City
            </div>
            <div>
              Position: 
              Midfielder
            </div>
            <div>
              Goals: 
              19
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Dominic Solanke
            </div>
            <div>
              Team: 
              Bournemouth
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              19
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Ollie Watkins
            </div>
            <div>
              Team: 
              Aston Villa
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              19
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Bryan Mbuemo
            </div>
            <div>
              Team: 
              Brentford
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              16
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Matheus Cunha
            </div>
            <div>
              Team: 
              Wolverhampton Wanderers
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              14
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Ryan Gravenberch
            </div>
            <div>
              Team: 
              Liverpool
            </div>
            <div>
              Position: 
              Midfielder
            </div>
            <div>
              Goals: 
              8
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Bruno Fernandes
            </div>
            <div>
              Team: 
              Manchester United
            </div>
            <div>
              Position: 
              Midfielder
            </div>
            <div>
              Goals: 
              12
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Kevin De Bruyne
            </div>
            <div>
              Team: 
              Manchester City
            </div>
            <div>
              Position: 
              Midfielder
            </div>
            <div>
              Goals: 
              10
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Morgan Gibbs-White
            </div>
            <div>
              Team: 
              Nottingham Forest
            </div>
            <div>
              Position: 
              Midfielder
            </div>
            <div>
              Goals: 
              10
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Anthony Gordon
            </div>
            <div>
              Team: 
              Newcastle United
            </div>
            <div>
              Position: 
              Midfielder
            </div>
            <div>
              Goals: 
              10
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Pascal Groß
            </div>
            <div>
              Team: 
              Brighton & Hove Albion
            </div>
            <div>
              Position: 
              Midfielder
            </div>
            <div>
              Goals: 
              8
            </div>
          </div>
          <div
            class="text-xl border-2 border-gray-400 rounded-md p-2 flex flex-col justify-center items-center cursor-default hover:text-white hover:bg-purple-500 hover:border-purple-500"
          >
            <div>
              Name: 
              Brennan Johnson
            </div>
            <div>
              Team: 
              Tottenham Hotspur
            </div>
            <div>
              Position: 
              Forward
            </div>
            <div>
              Goals: 
              9
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
